internal/etcd/etcd.go

```diff
@@ -149,6 +151,74 @@ func (c *ClientEtcdAPI) Close() error {
 	return nil
 }
 
+// UsedSpacePercentage returns the used space as a percentage of the storage quota
+func (c *ClientEtcdAPI) UsedSpacePercentage(ctx context.Context, member Member) (int, error) {
+	const (
+		storageQuotaKey = "etcd_server_quota_backend_bytes"
+		storageSizeKey  = "etcd_mvcc_db_total_size_in_bytes"
+	)
+	if len(member.ClientURLs) < 1 {
+		return 0, fmt.Errorf("unable to get used space, etcd member has no client urls")
+	}
+	url := member.ClientURLs[0] + "/metrics"
+	resp, err := http.Get(url)
+	if err != nil {
+		return 0, fmt.Errorf("failed to get metrics from etcd: %w", err)
+	}
+
+	b, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return 0, fmt.Errorf("failed to read metrics returned from etcd: %w", err)
+	}
+	defer resp.Body.Close()
+
+	strs := strings.Split(string(b), "\n")
+	var metricsWeCareAbout []string
+	for _, str := range strs {
+		if !strings.Contains(str, "HELP") &&
```

Comment:
Random metric from a node_exporter:
```
# HELP go_memstats_heap_objects Number of allocated objects.
# TYPE go_memstats_heap_objects gauge
go_memstats_heap_objects 9124
```
Could you not simplify this IF by doing 
```
if strings.HasPrefix(str, "go_memstats_heap_objects") || ...
```
And that way remove the two `!strings.Contains(str, "TYPE"`?